charts/document-engine/values.yaml

```diff
@@ -10,91 +10,79 @@ image:
-        name: ""
-        # Key names
-        apiTokenKey: API_AUTH_TOKEN
-        jwtPublicKeyKey: JWT_PUBLIC_KEY
-        jwtAlgorithmKey: JWT_ALGORITHM
-    # Document Engine Dashboard settings
-    dashboard:
-      # Enable dashboard
-      enabled: true
-      # `DASHBOARD_USERNAME`
-      username: admin
-      # `DASHBOARD_PASSWORD` — will generate a random password if not set
-      password: ""
-      # Use an external secret for dashboard credentials
-      # instead of the values from `pspdfkit.auth.dashboard.*`
-      externalSecret:
-        # External secret name
-        name: ""
-        # Key names
-        usernameKey: DASHBOARD_USERNAME
-        passwordKey: DASHBOARD_PASSWORD
+documentEngineLicense:
+  # Activation key for online activation (most common) or
+  # license key for offline activation.
+  # Results in `ACTIVATION_KEY` environment variable
+  # or `LICENSE_KEY` (if `isOffline` is `true`)
+  activationKey: ""
+  # Is the license offline: set to `true` if
+  # an offline license key was provided in `activationKey`
+  isOffline: false
+  # Query existing secret for the activation key
+  externalSecret:
+    # External secret name for license
+    name: ""
+    # The key in the secret used to retrieve the activation key
+    key: DOCUMENT_ENGINE_ACTIVATION_KEY
+
+# Document Enging API authentication
+apiAuth:
+  # `API_AUTH_TOKEN` should be long enough
+  apiToken: secret
+  jwt:
+    # Enable JWT
+    enabled: false
+    # `JWT_PUBLIC_KEY`
+    publicKey: none
+    # `JWT_ALGORITHM`
+    # Supported algorithms: RS256, RS512, ES256, ES512.
+    # See RFC 7518 for details about specific algorithms.
+    algorithm: RS256
   # A string used as the base key for deriving secret keys for the purposes of authentication.
   # Choose a sufficiently long random string for this option.
   # To generate a random string, use: `openssl rand -hex 256`.
   # This will set `SECRET_KEY_BASE` environment variable.
-  secretKeyBase:
-    value: ""
-    # Alternatively, provide an external secret
-    externalSecret:
-      # External secret
-      name: ""
-      # Key in the secret
-      key: SECRET_KEY_BASE
-  # Log level
-  log:
-    level: info
+  secretKeyBase: ""
+  # Use an external secret for API credentials
+  externalSecret:
+    # External secret name
+    name: ""
+    # Key names
+    apiTokenKey: API_AUTH_TOKEN
+    # If external secret is enabled, but `jwtAlgorithmKey` is not set,
+    # the algorithm will be retrieved from the `apiAuth.jwt.algorithm` value
+    jwtAlgorithmKey: JWT_ALGORITHM
+    # If external secret is enabled, but `jwtPublicKeyKey` is not set,
+    # the public key will be retrieved from the `apiAuth.jwt.publicKey` value
+    jwtPublicKeyKey: JWT_PUBLIC_KEY
+    # If external secret is enabled, but `secretKeyBaseKey` is not set,
+    # the secret key base will be retrieved from the `apiAuth.secretKeyBase` value
+    # or generated automatically
+    secretKeyBaseKey: SECRET_KEY_BASE
+
+config:
```

Comment:
This is a bit strange top level name. What kind of "config" is this section related to? Looks like DE misc configuration.

Example split into more logical sections: 

```
coreWorker:
   poolSize: 16
   readAnnotationBatchTimeoutSeconds: 20
   # Technically it's handled by core, but don't really like to have it here. Maybe something where we group which features are available (other candidates for that are in `limits`)?
   automaticLinkExtraction: true
   
documentEngineTimeouts:
   requestTimeoutSeconds: 60
   workerTimeoutSeconds: 60
   generationTimeoutSeconds: 60
   urlFetchTimeoutSeconds: 5

limits:
    maxUploadSizeMegaBytes: 950
    allowDocumentUploads: true
    allowRemoteDocuments: true
    # Maybe these could be even grouped under generation?
    allowDocumentGeneration: true
    allowRemoteAssetsInGeneration: true

asyncJobs: 
   jobsTtlSeconds: 172800
   # Possibly stuff like `backend: built-in` if we want to make the jobs implementation more backend agnostic (e.g. not require Oban in Postgres only).

network: 
   proxy: 
      http: ""
      https: ""

replaceSecretsFromEnv: true
```